demo/src/index.ts

```diff
@@ -3,7 +3,7 @@ import { TracingInstrumentation } from '@grafana/agent-tracing-web';
 import { initializeGrafanaAgent, getWebInstrumentations } from '@grafana/agent-web';
 
 initializeGrafanaAgent({
-  url: 'http://localhost:12345/collect',
+  url: '/collect',
```

Comment:
wut, same host? :thinking:

Reply:
For the time being, the demo also requires the receiver to be installed locally but that's not necessary unless you want to really check the data somewhere (and most of the times I don't as I just check the payloads) that are sent via `fetch`.

I added a proxy configuration (see `.proxyrc.js`) that responds with `200` to all the requests.

Reply:
I think we can improve this a lot when we refactor the demo 🤔 .